spec.json

```diff
@@ -344,6 +346,7 @@
         },
         "largeMotor": {
             "friendlyName": "Large Motor",
+            "systemDeviceNameConvention": "motor{0}",
```

Comment:
In python this gets translated to file system mask, so `*` may be used directly. `{0}` is also not a problem though, because it coincidentally is the string format marker in python (as in `'motor{0}'.format('*')`).